feed-integration-webapp/src/main/java/org/bahmni/module/feedintegration/crater/CraterAPIcalls.java

```diff
@@ -0,0 +1,158 @@
+package org.bahmni.module.feedintegration.crater;
+
+import org.apache.http.HttpEntity;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpUriRequest;
+import org.apache.http.client.methods.RequestBuilder;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+
+import org.bahmni.module.feedintegration.atomfeed.contract.patient.OpenMRSPatientFullRepresentation;
+import org.bahmni.module.feedintegration.atomfeed.contract.patient.OpenMRSPatientIdentifier;
+import org.bahmni.module.feedintegration.atomfeed.contract.patient.OpenMRSPersonName;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.Optional;
+
+import static com.sun.org.apache.xml.internal.utils.LocaleUtility.EMPTY_STRING;
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+public class CraterAPIcalls {
+
+    private final CraterProperties properties = CraterProperties.getInstance();
+    CraterLogin craterLogin = new CraterLogin();
+    private final String auth = craterLogin.getToken();
+
+    private static final Logger logger = LoggerFactory.getLogger(CraterAPIcalls.class);
+
+    public CraterAPIcalls() {
+    }
+
+    public HttpURLConnection getConnection(String api, String httpMethod, String auth) throws IOException {
+        URL url = new URL(properties.getUrl() + "/api/v1/" + api);
+        HttpURLConnection con = (HttpURLConnection) url.openConnection();
+        con.setRequestMethod(httpMethod);
+        con.setRequestProperty("Authorization", "Bearer " + auth);
+        con.setRequestProperty("Content-Type", properties.getContent_type());
+        con.setRequestProperty("Accept", properties.getAccept());
+        con.setRequestProperty("company", properties.getCompany());
+        con.setDoOutput(Boolean.parseBoolean(properties.getSetDoOutput()));
+        con.setDoInput(Boolean.parseBoolean(properties.getSetDoInput()));
+        return con;
+    }
+
+    public String login_verification(String auth) throws Exception {
```

Comment:
Why does this method return Strings, instead of a type or boolean? 
I would recommend to choose boolean to indicate login successful/failed, or use an enum.

Reply:
Changed it to boolean.